testdata/request.txt

```diff
@@ -0,0 +1,73 @@
+# Tests a request with a referenced request body
+
+openapi-httprequest api.yaml
+cmp api-params.go expect-api-params.go
+
+-- api.yaml --
+openapi: 3.0.0
+info:
+  description: test service description
+  title: test service
+  version: 1.0.0
+paths:
+  /test:
+    post:
+      description: Post test data (description)
+      operationId: PostTestData
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              $ref: '#/components/schemas/PostRequest'
```

Comment:
This doesn't look valid to me according to https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#referenceObject.
The `$ref` property needs to be on its own.